src/seedu/addressbook/AddressBook.java

```diff
@@ -208,7 +213,18 @@
 
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        if (args.length >= 2) {
```

Comment:
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

A single PR should have a single activity. For this week this is excused. However, from the next tutorial onwards please have separate PRs for each activity.